### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/vmware/vmw-ref-vcd97-known-issues.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+### Cannot select NIC type when adding a new NIC
+
+#### Description
+
+When adding a new NIC in the *Hardware* section of the VM details page, it is not possible to specify the NIC type.
+
+#### Solution
+
+<Is there a solution for this one?>
```
Comment (SteveHallUKCLoud) [inline articles/vmware/vmw-ref-vcd97-known-issues.md]: still no answer?

File: articles/vmware/vmw-ref-vcd97-known-issues.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Make sure to scroll through the list of operating systems to find the one you want to use for your VM.
+
+## Known issues with edge gateways
+
+### Edge with VPN shows as working when it is not
+
+#### Description
+
+Status icon for edge does not change even though there are issues with IPsec VPN or load balancing
```
Comment (SteveHallUKCLoud) [inline articles/vmware/vmw-ref-vcd97-known-issues.md]: no full stop

File: articles/vmware/vmw-ref-vcd97-known-issues.md (hunk 1 above)
Comment (suehighmoor) [reply articles/vmware/vmw-ref-vcd97-known-issues.md]: @mikepukcloud, as part of your review, could you confirm if there is a solution/workaround for this issue? Thanks!

File: articles/vmware/vmw-ref-vcd97-known-issues.md (hunk 2 above)
Comment (suehighmoor) [reply articles/vmware/vmw-ref-vcd97-known-issues.md]: Fixed

File: articles/vmware/vmw-ref-vcd97-known-issues.md (hunk 1 above)
Comment (suehighmoor) [reply articles/vmware/vmw-ref-vcd97-known-issues.md]: Add a solution of:
There is currently no workaround for this issue. We have raised a bug with VMware.

**Comment by suehighmoor** [comment]
@SteveHallUKCLoud, I've addressed your comments. Could you take a quick look please?

**Pull request merged**